### PR TITLE
Space Traveller Hub React Project

### PR DESCRIPTION
**Pull request by kessio**
Features implemented:
Config & basic setup
Created React App
Installed [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start)
Downloaded the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo
Created routes and view components (Rockets, Missions, My Profile, Dragons [only if your team has 3 members])
Used for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link)
Created directories for all Redux state slice files
Website features
The Rockets section displays a list of all available SpaceX rockets
Implemented functionality on buttons so users can book each rocket by clicking the reservation button or cancel the previously made booking
The Missions section displays a list of current missions along with their brief description and participation status
Added a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier
Used useSelector() Redux Hook to select the state slices and render lists of rockets and missions in corresponding routes
Made Rockets is the default view, so you must fetch rockets data when the application starts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
```
Comment (rloterh) [inline package.json:1]: - [ ] [Redux Logger](https://www.npmjs.com/package/redux-logger) is missing from your package. Kindly note that this activity requires that you make use `redux-logger` package. Kindly ensure to add/use the same.

File: src/__tests_/MissionTable.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { create } from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import store from '../redux/configureStore';
+import MissionTable from '../components/missions/MissionTable';
+
```
Comment (rloterh) [inline src/__tests_/MissionTable.test.js:8]: - [ ] There are some failing test cases. Kindly ensure that **all** test cases are passing.

**Failing test:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mukumbuta & Sharon, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Implemented required components
- State management with redux is properly done
- Data is properly fetched from the endpoint via store
- Navigations work as intended
- Reviewed projects in turns

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

**You can also consider:**
- Installed bootstrap package was not working correctly, however, bootstrap was working correctly after reinstalling the bootstrap package using the command `npm install bootstrap --save`
- Generally, the subheading text Author1 and Author2 should be replaced with your respective full names in the ReadMe.
-  It's recommended that you add an MIT license or any other similar license to your ReadMe.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
Hi Team,

 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**